doc/motion_planning_api/motion_planning_api_tutorial.rst

```diff
@@ -16,20 +11,20 @@ Getting Started
 ---------------
 If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
 
+This tutoral has made use of **xterm** and a simple prompter to help the user progress through each demo step.
+To install xterm please run the following command: ::
+
+   sudo apt-get install -y xterm
```

Comment:
This is no longer necessary, please convert this to use the rvizvisualtools prompt?